src/plugins/condition/components/Criterion.vue

```diff
@@ -162,6 +161,18 @@ export default {
                 }
             }
             return type;
+        },
```

Comment:
You shouldn't need this anymore. Let's revert this change and put the other changes back that aren't related to the id

Reply:
done